apps/account/templates/a4_candy_account/user_agreements.html

```diff
@@ -0,0 +1,44 @@
+{% extends "a4_candy_account/account_dashboard.html" %}
+
+{% load i18n static thumbnail %}
+
+{% block title %}{% trans 'User Agreements' %} &mdash; {{ block.super }}{% endblock %}
+{% block dashboard_content %}
+    <h1 class="mt-0">{% trans 'User Agreements' %}</h1>
+    <p>{% trans 'Here you can edit all your agreements' %}</p>
+
+    <form method="post">
+        {% csrf_token %}
+
+        {{ formset.management_form }}
+        {% for agreement_form in formset.forms %}
+        <div class="d-flex bg--bottom-border">
+            <div class="pe-4">
+                {% if agreement_form.instance.organisation.logo %}
+                <img src="{% thumbnail agreement_form.instance.organisation.logo '0x160' %}" alt="" class="account__org-image" />
```

Comment:
@phillimorland the image still shrinks in width, even after I added the width. I only managed to fix that if I wrap the image in a div. Is that ok to do? Or is there a class that makes an inline-block behave like a div? I tried `d-block`, but that did not work.

Reply:
@fuzzylogic2000  in this case actually I think it's about flex item inheritance, the direct children of a flex parent can be manipulated but once there is a structural layer between flex has no effect so I think it's fine. (I hope that's what's happening :p)

Reply:
@phillimorland Ah, so I have to add the div to make the image not shrink? But why does the div without the image, but only blue bg not shrink?

Reply:
oh that I don't know, does it still do it when we use a normal image with display: block and height and width, not a thumbnail tag? asking as the inline styles will always override the loaded ones so maybe that's why?

Reply:
also you can try flex-shrink: 0 on the image

Reply:
adding the div was the only way, I tried everything else. So, here it is. Another div around the image, which was in an extra div anyway. :woman_shrugging: